app/src/main/java/cloudcity/dataholders/Iperf3RunnerData.java

```diff
@@ -0,0 +1,80 @@
+package cloudcity.dataholders;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+import java.sql.Timestamp;
+import java.util.HashMap;
+
+import cloudcity.Iperf3Monitor;
+import de.fraunhofer.fokus.OpenMobileNetworkToolkit.Iperf3.Iperf3Fragment;
+
+/**
+ * A simple POJO/mule class to transfer data between <br>
+ * {@link Iperf3Monitor#startDefault15secTest()}<br>
+ * and <br>
+ * {@link cloudcity.Iperf3Monitor#startIperf3Test(Context, Iperf3RunnerData)}<br>
+ * methods
+ */
+public class Iperf3RunnerData {
+
+    @NonNull
+    private final String joinedCommand;
+    @NonNull
+    private final String[] commandList;
+    @NonNull
+    private final HashMap<String, String> dataMap;
+    @NonNull
+    private final HashMap<String, Boolean> booleanMap;
+    @NonNull
+    private final Timestamp timestamp;
+    @NonNull
+    private final Iperf3Fragment.Iperf3Input stupidInput;
```

Comment:
I wanted to use some other S word, but that one is unprofessional. this one is stupid because:
- it's embedded in the database schema
- we have to use it, because or iperf3 'reading' mechanism relies on some parts of it
- it's coupled to the iperf3 test
- it has hardcoded UI elements in itself which are also used throughout the original program; just take a look at Iperf3Fragment and all the other four Iperf3 things.
- to top it all of, why couldn't it have been extracted as a separate class? Because the outer class is touching alot of things in this inner class directly throughout it's codebase, and the inner class is touching alot of Iperf3Fragment's things as well. So it's also kinda tightly coupled to it's enclosing class, for no real, good and justfyable reason.

Will not change.